http/src/request/channel/message/create_message.rs

```diff
@@ -26,8 +26,9 @@ use twilight_model::{
         message::{AllowedMentions, MessageReference},
         Message,
     },
-    id::{ChannelId, MessageId},
+    id::{ChannelId, MessageId, StickerId},
 };
+use crate::request::channel::message::create_message::CreateMessageErrorType::TooManyStickers;
```

Comment:
i believe this import is unnecessary

Reply:
I think my IDE was being a bit aggressive there.  Will remove.